### PR TITLE
Feat: Enable automation support for VoiceOver/Safari/MacOS

### PR DESCRIPTION
**Pull request by gnarf**

### CODE REVIEW

**Review by howard-e — CHANGES_REQUESTED** [review]
Confirmed this allows VO and Safari tests to be queued up and ran as expected. Great stuff!

I left a minor question inline about a `logging` variable.

However, the main reason I think a 2nd look should be had here is because the prompt to trigger automation when an eligible report is created seems to be missing. I would have expected to be able to simply select that when creating a `VoiceOver` report now. In fact, it is also missing for when I'm creating an NVDA report as well. I confirmed it still popped up on `development` as shown in the screenshot but didn't on this branch.

screenshot showing option to run a newly created test plan report with a bot

File: server/models/services/ModelService.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ const getById = async (
  */
 const getByQuery = async (
     model,
-    { where, attributes = [], include = [], transaction }
+    { where, attributes = [], include = [], logging, transaction }
```
[the remark this answers is not in the document]
Comment (gnarf) [reply server/models/services/ModelService.js:74]: it's not, but it's clever - if you having trouble figuring out why the SQL query isn't working passing `logging: console.log` will dump the build SQL query - I can remove it but I wanted to point it out - glad you caught it

**Comment by gnarf** [comment]
> However, the main reason I think a 2nd look should be had here is because the prompt to trigger automation when an eligible report is created seems to be missing. I would have expected to be able to simply select that when creating a `VoiceOver` report now. In fact, it is also missing for when I'm creating an NVDA report as well. I confirmed it still popped up on `development` as shown in the screenshot but didn't on this branch.

Nice catch! I can take a quick look, but it's likely that the test report isn't passing the `key` property for at/brower -- Any chance you could give me more direct instructions where that popup shows up and I'll try to carve out 30 minutes a little later today to hunt this down.

**Comment by howard-e** [comment]
> Any chance you could give me more direct instructions where that popup shows up and I'll try to carve out 30 minutes a little later today to hunt this down.

@gnarf sure! On `development`, it pops up anytime I'm creating a Test Plan Report which uses NVDA as the AT. So in 2 cases currently:
1. On Test Queue page, as an admin, use the `Add Test Plans to the Test Queue` section to add a report to the queue which uses `NVDA` for the AT option.
2. On the Data Management page, as an admin, using the Test Plan Report status dialog pop up to add any report which is missing and uses the `NVDA` AT.

**Comment by gnarf** [comment]
https://github.com/w3c/aria-at-app/pull/1101/commits/a659c8374e004358a5d44dacdc05e06249e83e67 should fix the issue with the add and run popup:

screenshot of interface correctly showing the add and run popup with voiceover and safari

File: server/models/services/ModelService.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (howard-e) [reply server/models/services/ModelService.js:74]: Yep you could remove it, but very helpful to note down!

**Review by howard-e — COMMENTED** [review]
Changes look good!

Although the test failure is quite puzzling to me (because I can't get it to fail locally). It may be worth figuring out but the assertion will have to be updated to `1` anyways because of #1097 changes recently being included in `development` that do away with the `<span class=sr-only>`.

**Comment by gnarf** [comment]
> Although the test failure is quite puzzling to me (because I can't get it to fail locally). It may be worth figuring out but the assertion will have to be updated to `1` anyways because of #1097 changes recently being included in `development` that do away with the `<span class=sr-only>`.

The test that was failing was on the merged branch, so it made sense I just needed to update for the new assumptions.

**Review by howard-e — APPROVED** [review]
Thanks for addressing the feedback!

**Pull request merged**